sql/src/main/java/io/crate/expression/scalar/arithmetic/CeilFunction.java

```diff
@@ -35,29 +35,40 @@
 
 public final class CeilFunction {
 
-    public static final String NAME = "ceil";
+    public static final String CEIL = "ceil";
+    public static final String CEILING = "ceiling";
 
     public static void register(ScalarFunctionModule module) {
-        module.register(NAME, new FunctionResolver() {
-            @Nullable
-            @Override
-            public List<DataType> getSignature(List<? extends FuncArg> funcArgs) {
-                return FuncParams.SINGLE_NUMERIC.match(funcArgs);
-            }
+        module.register(CEIL, new CeilFunctionResolver(CEIL));
+        module.register(CEILING, new CeilFunctionResolver(CEILING));
+    }
+
+    private static class CeilFunctionResolver implements FunctionResolver {
```

Comment:
lgtm
 
Just a minor note, maybe we can use `BaseFunctionResolver` to remove a bit of code like the `getSignature` method override and the first condition in the `getForTypes` method

Reply:
Yep I was considering it, but felt like it would go too far within the same commit. I might follow up later.